lunatrace/bsl/backend/src/models/scan.ts

```diff
@@ -81,9 +81,9 @@ export async function runLunaTraceScan(sbomStream: Readable): Promise<string> {
       outputBuffers.push(Buffer.from(chunk));
     });
     lunatraceCli.stderr.on('data', (errorChunk) => {
-      // log.warn('lunatrace cli stderr', {
-      //   output: errorChunk.toString(),
-      // });
+      log.warn('lunatrace cli stderr', {
```

Comment:
was this intentional?

Reply:
Yep, I had disabled it because of the annoying spam from 401s in dev.